webapp/src/main/java/com/box/l10n/mojito/service/pullrun/PullRunService.java

```diff
@@ -46,8 +46,9 @@ public PullRun getOrCreate(String pullRunName, Repository repository) {
   }
 
   public void deleteAllPullEntitiesOlderThan(Duration retentionDuration) {
-    DateTime beforeDate = DateTime.now().minusSeconds((int) retentionDuration.getSeconds());
-    Timestamp sqlBeforeDate = new Timestamp(beforeDate.toDate().getTime());
+    ZonedDateTime beforeDate =
+        ZonedDateTime.now().minusSeconds((int) retentionDuration.getSeconds());
```

Comment:
wondering why `(int)` here. it is in the old so it should probably kept for now ... but maybe add a comment.

Reply:
the previous method from joda.time was expecting an int but the java.time can handle long values. We no longer need the (int) casting. Fixed, thanks for flagging